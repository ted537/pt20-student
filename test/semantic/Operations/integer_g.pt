using output;
val int1 = 5;
val int2 = 6;

if (int2 > int1) then
	put(1);
end;
