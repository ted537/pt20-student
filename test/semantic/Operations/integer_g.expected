   .tFileDescriptor
   .tLiteralInteger
   oEmitValue
   % value emitted 2
   .tFileBind
   .tLiteralAddress
   oEmitDataAddress
   % value emitted 0
   .tStoreInteger
   .tIfBegin
       .tLiteralInteger
       oEmitValue
       % value emitted 6
       .tLiteralInteger
       oEmitValue
       % value emitted 5
      .tGT
   .tIfThen
   oEmitNullAddress
   % value emitted -32767
      .tWriteBegin
        .tTrapBegin
        .tLiteralAddress
        oEmitValue
        % value emitted 0
        .tVarParm
        .tParmEnd
          .tLiteralInteger
          oEmitValue
          % value emitted 1
        .tParmEnd
         .tLiteralInteger
         oEmitValue
         % value emitted 10
         .tParmEnd
         .tTrap
         oEmitTrapKind(trWriteInteger)
         % value emitted 8
       .tWriteEnd
   .tIfEnd
 .tTrapBegin
 .tTrap
 oEmitTrapKind(trHalt)
 % value emitted 0
